protostar/commands/test/environments/standard_test_execution_environment.py

```diff
@@ -0,0 +1,14 @@
+from protostar.commands.test.environments.test_execution_environment import (
+    TestExecutionEnvironment,
+    TestExecutionResult,
+)
+
+
+class StandardTestExecutionEnvironment(TestExecutionEnvironment):
+    def __init__(self, *args, profiling=False, **kwargs):
+        self.profiling = profiling
+        super().__init__(*args, **kwargs)
+
+    async def execute(self, function_name: str) -> TestExecutionResult:
+        self.set_profile_flag(self.profiling)
```

Comment:
`set_profile_flag` is defined in the `ExecutionEnvironment`. IMO that method should be moved to this class or this class should be removed.

The class name is also vague. I'd name it `ProfilableTestExecutionEnvironment` but we should composition over inheritance.

Maybe `TestExecutionProfiler` passed as a dependency to `TestExecutionEnvironment` would be better approach.